### ISSUE TITLE
Do file operations check not in the configuration phase

### ISSUE THREAD

**Issue by emartynov** [runningcode/gradle-doctor#46]
I think after the latest fix with the symlink you introduced also performance issue - the file operations are checked during the gradle configuration that is not recommended behaviour.

How about moving all checks to the execution phase?

**Comment by runningcode** [comment]
Good idea. How can we do that? We aren't configuring tasks and we want to fail the build as soon as possible.
